docs/guides/basics/parameters/index.md

```diff
@@ -72,6 +72,67 @@ The order that you define the parameters determines the order they are passed in
+
+String parameters are compatible with almost every column type in Coda, as most have a text representation. At times a string parameter may be better than a more semantically accurate type, as it allows you to access the value as shown to the user.
+
+
+### Rich text
+
+Use the `Html` parameter type to pass text values with formatting included. Coda will convert the formatting to an equivalent block of HTML markup, and pass it to the `execute` function as a [JavaScript String][mdn_string].
+
+Like string parameters, HTML parameters can accept a wide array of column types. The generated HTML may be quite different than how it displays in Coda however, and is closer to what you'd get if you pasted that value into another rich text editor.
+
+!!! warning "HTML markup may change"
+    The generated HTML for a given value is not a stable API surface that you should rely on. We may change it at any time without warning, so we don't recommend that you parse it to extract information. Use it for display purposes only.
+
+
+### Numbers
+
+Use the `Number` parameter type to pass a number to your formula. Coda will automatically apply the [`ToNumber()`][ToNumber] formula to the input and pass it to the `execute` function as a [JavaScript Number][mdn_number].
+
+The number equivalent for some column types may not be obvious. Specifically:
+
+- **Percent** values will be converted into the equivalent fraction. For example, "75%" will be passed as `0.75`.
+- **Date** and **Date and time** values will be converted into the number of days since 1899-12-30[^1]. For example, "1955-11-12" will be passed as `20405`.
+- **Time** and **Duration** values will be converted into a number of days. For example, "12 hrs" will be passed as `0.5`.
+
+
+### Booleans
+
+Use the `Boolean` parameter type to pass a boolean (true/false) to your formula. Coda will pass the value to the `execute` function as a [JavaScript Boolean][mdn_boolean].
+
+
+### Dates
+
+Use the `Date` parameter type to pass a date value to your formula. Coda will automatically apply the [`ToDateTime()`][ToDateTime] formula to the input and pass it to the `execute` function as a [JavaScript Date][mdn_date].
+
+JavaScript Date objects can only represent a specific moment at time. This means that they can't easily represent less specific concepts like a day (regardless of time), a time (regardless of day), or duration. Coda handles those column types using the following logic:
+
+- **Date** values will be converted into a datetime representing midnight on that day in the document's timezone.
+- **Time** and **Duration** values will be converted a datetime that is that much time past midnight on 1899-12-30[^1], in the document's timezone. For example, "12 hours" in a document set to UTC will be passed as `1899-12-30T12:00:00.000Z`.
```

Comment:
Worth adding some practical tips on how you might work with such values? E.g. "To extract just the data component again, call date.getXYZ()...". or is that what the TBD guide content is for.

Reply:
Ya, I was planning to put that in the TBD guide.